modules/jpa-envers/src/main/java/io/oasp/module/jpa/dataaccess/base/AdvancedRevisionEntity.java

```diff
@@ -15,8 +15,6 @@
 import org.hibernate.envers.RevisionNumber;
 import org.hibernate.envers.RevisionTimestamp;
 
-import io.oasp.module.jpa.dataaccess.api.AdvancedRevisionListener;
```

Comment:
Yes, perfect the deprecated legacy support version of AdvancedRevisionListener is now used here. Correct!